src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Mappers/Apprentice/RecognisePriorLearningViewModelToResultMapper.cs

```diff
@@ -79,4 +81,43 @@ public async Task<RecognisePriorLearningResult> Map(PriorLearningDetailsViewMode
             };
         }
     }
+
+    public class PriorLearningDataViewModelToResultMapper : IMapper<PriorLearningDataViewModel, RecognisePriorLearningResult>
+    {
+        private readonly IOuterApiService _outerApiService;
+        private readonly ICommitmentsApiClient _commitmentsApiClient;
+
+        public PriorLearningDataViewModelToResultMapper(IOuterApiService outerApiService, ICommitmentsApiClient commitmentsApiClient)
+        {
+            _outerApiService = outerApiService;
+            _commitmentsApiClient = commitmentsApiClient;
+        }
+
+        public async Task<RecognisePriorLearningResult> Map(PriorLearningDataViewModel source)
+        {
+            var update = _outerApiService.PriorLearningData(source.CohortId, source.DraftApprenticeshipId,
+                new CreatePriorLearningDataApimRequest
+                {
+                    DurationReducedBy = source.DurationReducedBy,
+                    CostBeforeRpl = source.CostBeforeRpl,
+                    DurationReducedByHours = source.DurationReducedByHours,
+                    IsDurationReducedByRpl = source.IsDurationReducedByRpl,
+                    PriceReducedBy = source.PriceReduced,
+                    TrainingTotalHours = source.TrainingTotalHours
+                }
+            );
+
+            var apprenticeship = _commitmentsApiClient.GetDraftApprenticeship(
```

Comment:
This is a direct call to the inner api and, since we already have call going to the outer api, we could just include all the information required in its response so that we don't need this.